modules/local/multiqc.nf

```diff
@@ -63,17 +52,18 @@ process MULTIQC {
     path "versions.yml"        , emit: versions
```

Comment:
Need to remove this line? https://github.com/JoseEspinosa/nf-core-chipseq/blob/865210afa074301fad8e83429dbae93b762ea0ae/modules/local/multiqc.nf#L46

Reply:
Done here 154674d9f939ee4195c0213ed2466ed996363d7f